ui/components/multichain/connected-accounts-menu/connected-accounts-menu.tsx

```diff
@@ -67,13 +95,30 @@ export const ConnectedAccountsMenu = ({
           <TsMenuItem
             iconName={IconName.SwapHorizontal}
             data-testid="switch-account-menu-item"
+            onClick={() => {
+              trackEvent({
+                category: MetaMetricsEventCategory.Navigation,
+                event: MetaMetricsEventName.NavAccountSwitched,
+                properties: {
+                  location: 'Main Menu',
+                },
+              });
+              dispatch(setSelectedAccount(identity.address));
+              onClose();
+              closeMenu();
```

Comment:
Let's change it to this
```suggestion
              dispatch(setSelectedAccount(identity.address));
              onClose();
              closeMenu();
```